corehq/apps/locations/views.py

```diff
@@ -221,17 +220,7 @@ def unarchive_location(request, domain, loc_id):
     })
 
 
-class EditLocationView(NewLocationView):
-    urlname = 'edit_location'
-    page_title = ugettext_noop("Edit Location")
-
-    @property
-    @memoized
-    def location_form(self):
-        if self.request.method == 'POST':
-            return LocationForm(self.location, self.request.POST)
-        return LocationForm(self.location)
-
+class IndividualLocationMixin(object):
```

Comment:
I guess this could be called `LocationMixin` as opposed to `LocationsMixin`.  Basically I wanted to pull some stuff out of `EditLocationView` specific to an individual location for re-use in `ProductsPerLocationView`.